### DISCUSSION TITLE
Gateway Object errors out "The column 'id' of the table wasn't found"

### DISCUSSION THREAD

**Discussion by Maverick3733** [migueesc123/PowerBIRESTAPI#158] [Q&A] [answered]
Great Connector for PBI API!!!

Starting to wade through the various objects, and ran into an issue.  For the Gateway query/object, it's throwing an error:

<img width="473" alt="ID NOT FOUND" src="https://user-images.githubusercontent.com/37269488/232251546-ae0b6b07-fa8f-4ad1-baa1-f92e6a2a6932.png">

Any thoughts?

**Comment by klinejordan** [comment, marked as answer]
Those queries only return data if you are a gateway admin. Even if you are a Tenant Admin, those will not return data unless you are an explicit admin of at least 1 gateway in your tenant
